build-tools/webpack/extract-manifest-plugin.js

```diff
@@ -54,79 +54,76 @@ class ExtractManifestPlugin {
 			//		window.__WEBPACK_MANIFEST.set('chunkhash', (chunkId) => {1:'abcd1234'}[chunkId]);
 			//
 			// - At runtime, when webpack wants to load the asset for the chunk #1, it will call `jsonpScriptSrc(1)` it will get back `my-name.abcd1234.js`
-			compilation.mainTemplate.hooks.assetPath.tap(
-				PLUGIN_NAME,
-				( filenameExpression, options, assetInfo ) => {
-					// When there is no `chunk.hashWithLength`, this hook has been called to generate the filename of an asset, not an expression.
-					// Return whatever Webpack generates unmodified.
-					if ( ! ( options.chunk && options.chunk.hashWithLength ) ) return filenameExpression;
+			compilation.hooks.assetPath.tap( PLUGIN_NAME, ( filenameExpression, options, assetInfo ) => {
```

Comment:
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#mainchunkmoduletemplate-deprecation

Hook moved to `Compilation`